Setting for optional handover (#2582)
spatialos/UnrealGDK: CHANGELOG.md

  NOTE: If your project does not use custom values for the `PositionUpdateFrequency` or `PositionDistanceThreshold`, then, by default, the updates will be sent with the same frequency as before and no action is required.
- Removed the `OnAuthorityLossImminent` Actor event.
- 'WorkerLogLevel' in Runtime Settings was split into two new settings - 'LocalWorkerLogLevel' and 'CloudWorkerLogLevel'. Update these values which will be set to 'Warning' by default.

> - Removed the method `RequiresHandoverData` from `UAbstractLBStrategy`, handover is now determined by the `USpatialMultiWorkerSettings` class by the `bHandoverEnabled` setting. If this is false, handover will never be evaluated.

Author: ```suggestion
```

- Allow specifying the locator port via `?locatorPort=` URL option when performing client travel.
- You can now enable/disable the multi-worker load balancing strategy with an in-editor toggle so that no Uasset files are changed. Select `Enable Multi-Worker` from the drop option from the `Start Deployment` button on the toolbar to use the multi-worker strategy or de-select to use a single worker strategy in the editor. The `Enable Multi-Worker` toggle in World Settings and the command line option  `-OverrideMultiWorker` have been removed as they are now redundant.
- Enabled packaging the command line arguments when building a mobile client by default.

> - Handover is now optional according to the load balancing strategy `RequiresHandoverData` method. 

Author: ```suggestion
- Handover is now optional according to the multi-worker settings (`USpatialMultiWorkerSettings`). 
```

> - Handover is now optional according to the multi-worker settings (`USpatialMultiWorkerSettings`). 

Author: ```suggestion
- Handover is now optional according to the multi-worker settings (`USpatialMultiWorkerSettings::bHandoverEnabled`). 
```

> - Handover is now optional according to the multi-worker settings (`USpatialMultiWorkerSettings::bHandoverEnabled`). 

Reviewer: defaults to true

- Allow specifying the locator port via `?locatorPort=` URL option when performing client travel.
- You can now enable/disable the multi-worker load balancing strategy with an in-editor toggle so that no Uasset files are changed. Select `Enable Multi-Worker` from the drop option from the `Start Deployment` button on the toolbar to use the multi-worker strategy or de-select to use a single worker strategy in the editor. The `Enable Multi-Worker` toggle in World Settings and the command line option  `-OverrideMultiWorker` have been removed as they are now redundant.
- Enabled packaging the command line arguments when building a mobile client by default.

> - Handover is now optional according to the multi-worker settings (`USpatialMultiWorkerSettings::bHandoverEnabled`). 

Author: ```suggestion
```
